### PR TITLE
Print warning and exit when config file doesn't exist

### PR DESCRIPTION
**Pull request by TheArcaneBrony**
Added a little bit of code to check that the config file doesn't exist. This prevents a crash when the config doesn't exist.

I've also added cmake-build-debug to the gitignore, since that directory contains build output when built from CLion.

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #include <templatebot/templatebot.h>
 #include <sstream>
+#define fs std::filesystem
```
Comment (raxyte) [inline src/main.cpp]: This is really bad, consider using a namespace alias instead.

**Review by raxyte — CHANGES_REQUESTED** [review]

**Comment by TheArcaneBrony** [comment]
I usually dont do C++, so my knowledge on best practices is nonexistant. Thanks for pointing this out!

File: src/main.cpp [hunk 2]
```diff
@@ -10,6 +11,11 @@ using json = nlohmann::json;
 
 int main(int argc, char const *argv[])
 {
+    if(!fs::exists("../config.json")) {
+        std::cout << canonical(fs::path("..")).append("config.json").string() << " does not exist!" << std::endl;
```
Comment (raxyte) [inline src/main.cpp:15]: Did you try to compile this? `canonical` is under `std::filesystem`, yet it's being used directly? Also, may I ask for your reasoning behind not just `std::cout << "../config.json doesn't exist\n";` and opting to use std::filesystem here?

Comment (TheArcaneBrony) [reply src/main.cpp:15]: Yes I did try to compile this. It builds and runs fine for me (see attached screenshot).
My reasoning for using canonical is to print the absolute file path, to reduce user confusion ("../config.json, but where is ..??").

image

**Review by raxyte — APPROVED** [review]

**Comment by braindigitalis** [comment]
std::filesystem is not really a good idea here, because on some systems (mainly GCC) it needs explicitly linking an extra library at link step, and the cmake hasn't been adjusted to do this where needed...

**Comment by TheArcaneBrony** [comment]
I'm not sure what I should be using instead?

**Pull request closed without merging**